module/support/configuration/GlobalConfig/data/config/darwin1/GlobalConfig.yaml

```diff
@@ -1,2 +1,2 @@
 playerId: 1
-teamId: 9
+teamId: 14
```

Comment:
Move this from a per robot config to a normal one. Let the merging handle switching IDs.